bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/BandsList.scala

```diff
@@ -10,9 +8,9 @@ import Scalaz._
  * It is used, e.g. to extract a magnitude from a target
  */
 sealed trait BandsList {
-  val bands: NonEmptyList[MagnitudeBand]
-  def extract(t: SiderealTarget) = bands.map(t.magnitudeIn).list.find(_.isDefined).flatten
-  def bandSupported(b: MagnitudeBand) = bands.list.contains(b)
+  val bands: List[MagnitudeBand]
```

Comment:
The reason this was a `NEL` is to indicate that it cannot be empty and I'd think that is a good thing. On the other hand this is the simplest solution to solve the serialization problem so this fix is ok
I have to wonder though how this class ends up being part of the model, it seems a bit removed to require serialization